src/components/EarnDashboard/components/PositionDetails/StakingPositionsByProvider.tsx

```diff
@@ -175,7 +175,7 @@ export const StakingPositionsByProvider: React.FC<StakingPositionsByProviderProp
           const opportunity = row.original
           const opportunityAssetId = opportunity.assetId
           const opportunityUnderlyingAssetId = opportunity.underlyingAssetId
-          const hasValue = bnOrZero(opportunity.fiatAmount).gt(0)
+          const hasValue = !bnOrZero(opportunity.fiatAmount).isZero()
```

Comment:
The third guts of this, changes the heuristics everywhere for staked crypto amount / fiat amount to handle negative amounts (doesn't touch rewards amounts checks since these can't be negative)